database/postgres/repo_count_test.go

```diff
@@ -162,3 +162,74 @@ func TestPostgres_Client_GetUserRepoCount(t *testing.T) {
 		}
 	}
 }
+
+func TestPostgres_Client_GetOrgRepoCount(t *testing.T) {
```

Comment:
I may have missed it but I think we want a test for the exclude functionality as well, right?